tests/end-to-end/infer.sh

```diff
@@ -17,45 +17,58 @@ echo ""
 atacworks denoise \
     --out_home $out_dir --label inference \
     --weights_path $expected_results_dir/model_latest/model_best.pth.tar \
-    --num_workers 0 \
-    --input_files $out_dir/no_label.h5 \
-    --intervals_file $out_dir/result.holdout_intervals.bed \
-    --sizes_file $ref_dir/hg19.auto.sizes \
-    --infer_threshold 0.5 \
+    --num_workers 0 --gpu 0 \
+    --noisybw $data_dir/HSC.5M.chr123.10mb.coverage.bw  \
+    --wg --interval_size 24000 \
```

Comment:
what's `wg` again? is there a longer name for it that can be more descriptive?

Reply:
It means whole genome. This option is removed in subsequent PRs. If user does not provide the label file or specify the regions, we automatically denoise the whole genome.

Reply:
okay great I'll resolve this